doc/index.rst

```diff
@@ -5,10 +5,9 @@
 
 .. container:: index-paragraph
 
-    Nilearn is a Python module for **fast and easy statistical learning on
-    NeuroImaging** data.
+    Nilearn is a Python module to enable **powerful and controlled statistical analysis of brain volumes**.
```

Comment:
But still also statistical learning tasks (classification, covariance learning, network extraction).

Reply:
Right, but this mission statement must be something very short.

Reply:
I think that I would remove controlled :)